spec/core/validators/validators.spec.tsx

```diff
@@ -23,11 +25,22 @@ describe('Input Validators', () => {
     expect(await Validators.EmptyString('')).toEqual({ emptyString: true });
     expect(await Validators.EmptyString(' ')).toEqual({ emptyString: true });
     expect(await Validators.EmptyString('a')).toEqual(null);
+    expect(await Validators.EmptyString([])).toEqual({ emptyString: true });
+    expect(await Validators.EmptyString(undefined)).toEqual({ emptyString: true });
   });
 
   it('should validate minlength correctly', async () => {
     expect(await Validators.MinLength(3)('a')).toEqual({ minlength: true });
     expect(await Validators.MinLength(3)('abcd')).toEqual(null);
+    expect(await Validators.MinLength(3)([])).toEqual({ minlength: true });
+    expect(await Validators.MinLength(3)(undefined)).toEqual({ minlength: true });
+  });
+  it('should validate maxLength correctly', async () => {
+    expect(await Validators.MaxLength(3)('a')).toEqual(null);
+    expect(await Validators.MaxLength(3)('')).toEqual(null);
+    expect(await Validators.MaxLength(3)([])).toEqual(null);
+    expect(await Validators.MaxLength(3)(undefined)).toEqual(null);
+    expect(await Validators.MaxLength(3)('abcd')).toEqual({ maxLength: true });
```

Comment:
Could you add an example when `maxLength: true` on array input

Reply:
done, also for the rest of the validators.